monitoring/backups_collector.go

```diff
@@ -0,0 +1,94 @@
+package monitoring
+
+import (
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+)
+
+type BackupsCollector struct {
+	*baseCollector
+
+	sizeMetric  metricInfo
+	stateMetric metricInfo
+}
+
+func NewBackupsCollector(
```

Comment:
Suggest aligning with other collector naming. `backupcollector` instead of `backupscollector`. The same for the file name.

Reply:
Fix in d156d1fdaa05700e384ff7ab0ea429f15ee27ab4